Pocket/pocket-ios
# IN-321: Add Tags to Save Extension

## Summary
Added the ability for users to add tags to a Saved Item via Save Extension

## References 
IN-321

## Implementation Details
* Move `AddTagsView` into the shared UI Component `Textile` and created a two view models (one for PocketKit and one for SaveTo Extension) which both conform to `AddTagsViewModel` protocol. Modified the `PocketSaveService` to be able to send the GraphQL mutations related to tags and update the saved item accordingly. Added a new `infoViewModel` called `.taggedItem` to show the user a confirmation view.
* Change the copy of existing item from `You've already saved this before. We'll move it to the top of your list.` to `You've already saved this. We'll move it to the top of your list.` as confirmed with product.
* Extended dismiss timer as it disappeared too quickly

## Test Steps
- [ ] Given that a user saves item to pocket using shared menu, 
and the user taps on Add Tags button, 
then the user should be able to view the Add Tags screen. 

- [ ] Given that sees the Add Tags view via shared extension, 
and the user interacts with it, 
then the behavior should be the same as general Add Tags feature.

- [ ] Given that a user saves a tag, 
then the user should view a confirmation page.

## PR Checklist:

## Screenshots

https://user-images.githubusercontent.com/6743397/195681326-4f7ac0f0-fa34-4fd8-aaa4-b420dc6553a6.mp4
